### PR TITLE
added project setup instructions

### PR DESCRIPTION
**Pull request by ZmagoD**
Hi,
I spent a lot of time with the setup for this project so I made a list of instructions "copy/paste" for setup this project. I used Cloud9 IDE as a development environment. I hope this will help someone!

Best regards

### CODE REVIEW

**Comment by theB264** [comment]
:+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,3 +26,104 @@ As described in `README.md`, we follow Agile methods and work closely with our c
...
+
+now, you should have PhantomJS properly installed on your system
+
+$ phantomjs --version
+
+ok nearly finished, now run the tests
+
+$ cd workspace
+$ bundle exec rspec spec/
+$ bundle exec cucumber
```
Comment (razorcd) [inline CONTRIBUTING.md]: You can run all the rspec and cucumber tests only by running `rake`.
Find more here:  https://github.com/AgileVentures/osra/blob/develop/CONTRIBUTING.md

**Comment by ZmagoD** [comment]
I just copied from AgileVentures http://www.agileventures.org/projects/osra-support-system/documents/getting-started-with-osra :) I will fix this. Thank you!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,3 +26,103 @@ As described in `README.md`, we follow Agile methods and work closely with our c
...
+
+now you have to setup the database and the permissions,
+follow these steps:
+
+$ sudo service postgresql start
+$ sudo sudo -u postgres psql
+
+now you are in postgresql console!
+
+postgres=# CREATE USER username SUPERUSER PASSWORD 'password';
```
Comment (gmunumel) [inline CONTRIBUTING.md]: Is this mandatory for Cloud9? 

To connect to postgres database we only use `postgres` as a user and with no password.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -26,3 +26,103 @@ As described in `README.md`, we follow Agile methods and work closely with our c
...
+in the config/database.yml file change the username, password and host for all environments:
+
+  username: <%= ENV['USERNAME'] %>
+  password: <%= ENV['PASSWORD'] %>
+  host: <%= ENV['IP'] %>
+  
+then head back to postgresql console and follow these steps:
+
+$ sudo sudo -u postgres psql
+postgres=# UPDATE pg_database SET datistemplate = FALSE WHERE datname = 'template1';
```
Comment (gmunumel) [inline CONTRIBUTING.md]: Are these steps necessary for Cloud9? Looks like a test of `postgres` functionality.

**Comment by ZmagoD** [comment]
when I try to start the postgres database as  shown here https://docs.c9.io/v1.0/docs/setting-up-postgresql  it throw me an permission error so I had to all these steps in order to make it work.

**Comment by theB264** [comment]
I've also had many stubborn problems with postgres permission errors both setting up osra and other projects.  I think adding this is a great idea.

**Comment by gmunumel** [comment]
I also had one recently. 
Great work @ZmagoD  :+1:

**Comment by ZmagoD** [comment]
Thank you!

**Pull request merged**